nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
 
 snippet:rabbitmq-config-basic
 
-The transport needs to connect the the RabbitMQ broker. By default the transport will look for a connection string called `NServiceBus/Transport` in the `app.config`.
+
+## Specifying the connection string
+
+The transport needs to connect to a RabbitMQ broker. By default, the transport will look for a connection string called `NServiceBus/Transport` in the `app.config`.
+
+To use a custom name for the connection string:
+
+snippet:rabbitmq-config-connectionstringname
+
+To specify the connection string in code:
+
+snippet:rabbitmq-config-connectionstring-in-code
 
 A typical connection string would look like this:
 
 snippet:rabbitmq-connectionstring
 
-In the above sample the transport is configured to connect to the RabbitMQ broker running at the machine `broker1`.
+In the above sample, the transport is configured to connect to a RabbitMQ broker running at the machine `broker1`.
+
+
+### Connection string options
 
 Below is the list of connection string options. Note that they are separated by a `;`.
 
 
+#### Host
+
+The host name of the broker.
+
+NOTE: This value is required.
+
+
 #### Port
 
-The port where the broker listens. 
+The port where the broker listens.
 
-Defaults: `5672`
+Default: `5672` or `5671` if `UseTls` is true
```

Comment:
Does this mean that if the "UseTls" setting is true then the value is either 5672 or 5671? This is not clear. 

How about: 

Default: `5672` if the `UseTls` setting is also set to `true`, otherwise the default value is `5671` (or something like that)